registry.json

```diff
@@ -1,3 +1,5 @@
 {
-  "dockstore" : "https://www.dockstore.org:8443"
+  "dockstore" : [
+      "https://www.dockstore.org:8443/api/v1/metadata"
+    ]
```

Comment:
If I recall correctly, the order is irrelevant. This is a just a way of pointing our index at a particular server (Dockstore in this case) multiple times if it supports different versions of the API.